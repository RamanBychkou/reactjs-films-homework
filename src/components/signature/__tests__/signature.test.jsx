import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Signature from '../signature';

test('Signature renders correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Signature />);
  const result = renderer.render(<Signature />);
  expect(result).toMatchSnapshot();
});
