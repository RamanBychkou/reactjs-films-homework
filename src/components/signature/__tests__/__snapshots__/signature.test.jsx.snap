// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signature renders correctly 1`] = `
<h1
  className="name"
>
  Roman Bychkou
</h1>
`;
